app/javascript/packs/application.js

```diff
@@ -7,3 +7,4 @@ require('chart.js')
 require('flatpickr')
 require('flatpickr/dist/l10n/ja')
 require("flatpickr/dist/themes/material_orange.css")
+require('solitude.js')
```

Comment:
次のように書かないと読み込まれないですね。

```suggestion
require('./solitude')
```